src/NuGetPackageVerifier/CompositeRules/SigningVerificationCompositeRule.cs

```diff
@@ -15,6 +15,7 @@ public class SigningVerificationCompositeRule : IPackageVerifierRule
             new AuthenticodeSigningRule(),
             new PowerShellScriptIsSignedRule(),
             new RequiredNuSpecInfoRule(),
+            new DefaultCompositeRule(),
```

Comment:
Scaffolding for instance: https://github.com/aspnet/Scaffolding/blob/dev/src/Microsoft.VisualStudio.Web.CodeGeneration/Properties/AssemblyInfo.cs#L7.